wgsl/index.bs

```diff
@@ -7647,7 +7647,7 @@ Note: Recall that the [=f32=] WGSL type corresponds to the IEEE-754 binary32 for
 When converting a floating point scalar value to an integral type:
 * If the original value is exactly representable in the destination type, then the result is that value.
 * If the original value has a fractional component, then it cannot be represented exactly in the destination type, and the result is rounded toward zero.
-* If the original value is out of range of the destination type, then TODO.
+* If the original value is out of range of the destination type, the result is the value in the destination type that is closest to the original value.
```

Comment:
is this how trivial casts work in the APIs?
seems like we'd need a mini-investigation here